app/controllers/internal/events_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
       @no_results = @events.blank?
 
       @success = params[:success]
+      @withdrawn = params[:success] == "withdrawn"
```

Comment:
I would maybe rename this param to be `status` or something, `success` doesn't really make sense with respect to the value